test/built-ins/Object/subclass-object-arg.js

```diff
@@ -20,5 +20,5 @@ class O extends Object {}
 var o1 = new O({a: 1});
 var o2 = Reflect.construct(Object, [{b: 2}], O);
 
-assert.sameValue(o1.a, 1);
-assert.sameValue(o2.b, 2);
+assert.sameValue(o1.a, undefined);
```

Comment:
Thank you for explanation: I got confused with "active function" term.
Perhaps we should add an assertion that `Object.getPrototypeOf(o{1,2})` is `O.prototype`?